### PR TITLE
P1 rating

### PR DESCRIPTION
**Pull request by vidhi-thakur**

### CODE REVIEW

File: components/component_style/button.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.btn-primary-outlined {
+    border: 2px solid var(--primary-color);
+    background-color: white;
+    color: var(--primary-color);
+}
+
+.btn-primary-outlined:hover {
+    background-color: #f7f7f7;
```
Comment (msdeshmukh009) [inline components/component_style/button.css:28]: This can also be a color variable as well.

**Review by msdeshmukh009 — APPROVED** [review]
All look good to me.

File: components/component_style/button.css (hunk 1 above)
Comment (vidhi-thakur) [reply components/component_style/button.css:28]: thank you for pointing out, updated this as variable but forgot to update here

**Review by shuklahimanshu81 — COMMENTED** [review]
Everything looks good to me. Ready to merge.

**Pull request merged**